### DISCUSSION TITLE
Keeps redirecting and cannot enter the GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by kroblus** [community/community#76955] [GitHub Education]
### Hi everyone,

Open the student package authentication address: https://education.github.com/discount_requests/application and it will automatically redirect to https://github.com/pricing. Student authentication cannot be completed.

**Comment by hackelite01** [comment]
Hello @kroblus ,  To troubleshoot, try clearing your browser cache and cookies, opening the authentication link in an incognito or private browsing window, or switching to a different browser.

**Reply by xiaoxiaojiujiu** [reply]
I have the same issue as him. Following your steps, it still redirects to https://github.com/pricing.

**Reply by hackelite01** [reply]
Kindly mail me all details if you want to get the pack approved.

**Reply by kroblus** [reply]
Of course I tried them all but it still didn't work

**Reply by hackelite01** [reply]
I can do it easily. Drop the details on mail at hackelite.sup@gmail.com

**Reply by fahmizainal17** [reply]
Could you guys help me get two answer accepted?

**Comment by ZheruiHuang** [comment]
Same issue. Had Tried all the above approaches but didn't work :(

**Reply by hackelite01** [reply]
https://github.com/orgs/community/discussions/76955#discussioncomment-7721816

**Comment by zihanghliu** [comment]
I have met the same problem. Has anyone found ways to fix it?

**Comment by white-z** [comment]
If your account was previously verified using an unrecognized educational institution, you may not be able to verify again.

**Comment by huskyhao** [comment]
same issue!!!!!!!!!!

**Comment by Kobe972** [comment]
Same issue. I opened a ticket and it is answered by a fucking robot with no useful information, and the ticket is closed. I can't reopen the ticket or submit a new ticket anymore (reopen: failed to submit the comment; new ticket: submitted successfully, but I can't see the ticket after refreshing the page)

**Comment by YahyaAzzam** [comment]
Create a new account, with removing the academic email from the first account and link it to the second account, the second account should be able to enter the application page.

**Reply by YahyaAzzam** [reply]
@kroblus @white-z @xiaoxiaojiujiu @Kobe972 @hackelite01 @ZheruiHuang @fahmizainal17 @HenryLiu0820 @XYCode-Kerman @huskyhao

**Reply by ycformal** [reply]
But I want to use the original account.

**Reply by heyewuyue1** [reply]
I tried this, but my second account also get to the same situation.

**Comment by Alexisliao10** [comment]
I have the same problem. I think it's the way GitHub Education bans 'bad actors' from entering the program ever again without explicitly stating it.

**Comment by heyewuyue1** [comment]
Same issue.
